let customersLoaded = false;

getCustomers = async () => {
    try {
    }
    catch(err){ console.log("getCustomers: ERROR", err); }
}


onSelectedCustomer = (event) => {
    try {
    }
    catch(err){ console.log("onSelectedCustomer: ERROR", err, event); }
}